config/dato/_query-api.mjs

```diff
@@ -0,0 +1,91 @@
+import axios from 'axios'
```

Comment:
Is there a reason why this is a `.mjs` file? In any case, it's very well written 👍🏼

Reply:
yeah, with `.mjs` you can use `import` and top-level await